blockchain/projects.go

```diff
@@ -55,21 +55,21 @@ type Projects struct {
 	accountsNumber int
 }
 
-// Reset the blockchain state.
-func (p *Projects) Reset(project *model.Project) ([]*model.Account, error) {
+// Reset the blockchain state and return the new number of accounts
+func (p *Projects) Reset(project *model.Project) (*int, error) {
```

Comment:
Why is int pointer returned? Wouldn't be better to just return new project model?

Reply:
We don't need to return new project models since they're only created when the client actually queries for them. I should change this to an int instead of int pointer though.